dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/view/AppView.java

```diff
@@ -48,14 +52,15 @@ public AppView(final AppDescriptor appDescriptor, final long configurationsCount
      * @param configurationsCount
      * @param sites
      */
-    public AppView(final AppDescriptor appDescriptor, final long configurationsCount, final List<SiteView> sites) {
+    public AppView(final AppDescriptor appDescriptor, final int configurationsCount, final List<SiteView> sites) {
         this.key = appDescriptor.getKey();
         this.name = appDescriptor.getName();
         this.description = appDescriptor.getDescription();
         this.iconUrl = appDescriptor.getIconUrl();
         this.allowExtraParams = appDescriptor.isAllowExtraParameters();
         this.configurationsCount = configurationsCount;
         this.sites = sites;
+        this.sitesWithWarnings = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assigning an Object to null is a code smell.  Consider refactoring.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5519512)